cmd/node/node.go

```diff
@@ -770,7 +768,15 @@ func (app *SpacemeshApp) getIdentityFile() (string, error) {
 }
 
 func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
-	log.Event().Info("Starting Spacemesh")
+	log.With().Info("Initializing Spacemesh", log.String("data-dir", app.Config.DataDir), log.String("post-dir", app.Config.POST.DataDir))
```

Comment:
why change this print? we rely on this text in our searches, you can add more parameters to it but I wouldn't change the main title